src/panels/lovelace/components/hui-action-editor.ts

```diff
@@ -53,51 +56,74 @@ export class HuiActionEditor extends LitElement {
     }
 
     return html`
-      <paper-dropdown-menu
-        .label="${this.label}"
-        .configValue="${"action"}"
-        @iron-select="${this._actionPicked}"
-      >
-        <paper-listbox
-          slot="dropdown-content"
-          attr-for-selected="value"
-          .selected=${this.config?.action ?? "default"}
+      <div class="dropdown">
+        <paper-dropdown-menu
+          .label=${this.label}
+          .configValue=${"action"}
+          @iron-select=${this._actionPicked}
         >
-          <paper-item .value=${"default"}>Default action</paper-item>
-          ${this.actions.map((action) => {
-            return html`<paper-item .value=${action}>${action}</paper-item>`;
-          })}
-        </paper-listbox>
-      </paper-dropdown-menu>
+          <paper-listbox
+            slot="dropdown-content"
+            attr-for-selected="value"
+            .selected=${this.config?.action ?? "default"}
+          >
+            <paper-item .value=${"default"}
+              >${this.hass!.localize(
+                "ui.panel.lovelace.editor.action-editor.actions.default_action"
+              )}</paper-item
+            >
+            ${this.actions.map((action) => {
+              return html`
+                <paper-item .value=${action}
+                  >${this.hass!.localize(
+                    `ui.panel.lovelace.editor.action-editor.actions.${action}`
+                  )}</paper-item
+                >
+              `;
+            })}
+          </paper-listbox>
+        </paper-dropdown-menu>
+        <ha-help-tooltip
+          .label=${"Default action is defaulted to the domain action."}
```

Comment:
Something like this?
```suggestion
          .label=${"The default action depends on the entity capabilities, it will be either toggled or the more info will be shown."}
```

Reply:
Translation?

Reply:
@iantrich Thats an old code line. I translated it the most recent commits